src/content/advanced/private-clusters/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Private clusters
+description: By default, the Kubernetes API endpoint is public and the cluster has internet access. This article shows which products offer private cluster features.
+weight: 35
+menu:
+  main:
+    parent: advanced
+user_questions:
+- How do I make a workload cluster private?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-06-13
+---
+
+By default, the Kubernetes API endpoint is public and the cluster has internet access. You can skip this article unless you plan on creating clusters with strictly-limited networking.
+
+The following products offer private cluster features:
+
+- CAPA (AWS EC2)
+- CAPZ (Azure VMs)
+
+Private clusters have special requirements:
+
+- Since a private workload cluster still has to be reachable by the management cluster, there must be networking peering between the two. This means that the IP ranges must not overlap.
+- Access to the Kubernetes API can be restricted to an internal network. It should then be reachable both by you as customer, and by Giant Swarm for support purposes. For example, a VPN can be used.
+- An HTTP proxy can be used to restrict which exact domains can be accessed at all. In this case, precise configuration is important or else the new cluster will not be able to fetch container images or reach other important resources such as internal communication to the Kubernetes API.
+
+To get everything set up correctly, please get in contact with Giant Swarm. We will help creating and supporting your desired private cluster configuration within your network architecture.
+
+At the moment, we have these – partially still hidden – [`kubectl gs template cluster` command line options]({{< relref "/use-the-api/kubectl-gs/template-cluster" >}}) to configure access to/from the cluster:
```

Comment:
I would avoid the phrase `partially still hidden` or I would add an explanation why